src/app/build/demo.js

```diff
@@ -35,4 +35,4 @@ object-assign
ation",patterns:[me,ve]},{name:"Features",patterns:[ye,ge,be,we]}],Ee=function(e){var t,n;function o(){var t;return(t=e.call(this)||this).state={env:"sandbox",errors:[]},t}n=e,(t=o).prototype=Object.create(n.prototype),t.prototype.constructor=t,t.__proto__=n;var a=o.prototype;return a.onChangeCode=function(e){this.setState({code:e,errors:[]})},a.onChangeEnv=function(e){this.setState({env:e})},a.onCodeRun=function(e){this.setState({errors:[]})},a.onCodeError=function(e){this.setState({errors:this.state.errors.concat(e.stack||e.toString())})},a.render=function(){var e=this,t=this.props.match.params.pattern||"client",n=r[t];n||(n=me);var o=this.state.env,a=document.body.getAttribute("data-base-url");return i.a.createElement("div",null,i.a.createElement(ce,{onChangeEnv:function(t){return e.onChangeEnv(t)}}),i.a.createElement("div",{className:"main"},i.a.createElement("div",{className:"column-left"},xe.map((function(e,t){return i.a.createElement("div",{key:t},i.a.createElement("h3",null,e.name),i.a.createElement("ul",null,e.patterns.map((function(e){return!e.nosidebar&&i.a.createElement(ae,{to:"/pattern/"+e.slug,key:e.slug,activeClassName:"active"},i.a.createElement("li",null,i.a.createElement("span",{className:"bullet"}),i.a.createElement("span",null,e.name)))}))))}))),i.a.createElement("div",{className:"column-middle"},i.a.createElement("div",{className:"intro"},i.a.createElement("h3",null,n.fullName),i.a.createElement("div",{className:"introp"},n.intro)),i.a.createElement("div",{className:"demo"},i.a.createElement("div",{className:"steps"},i.a.createElement("div",{className:"step right"},"1. Edit the code"),i.a.createElement("div",{className:"step bottom"},"2. Try the button"),this.state.errors.length?i.a.createElement("div",{className:"errors"},this.state.errors.map((function(e){return i.a.createElement("p",{key:e},e)}))):i.a.createElement(he,{setup:n.setup,pattern:t,code:this.state.code,onError:function(t){return e.onCodeError(t)}}),i.a.createElement("div",{className:"step right"},"3. Copy code to your site!")))),i.a.createElement("div",{className:"column-right"},i.a.createElement(fe,{code:n.code({env:o,baseURL:a}),onChange:function(t){return e.onChangeCode(t)}}))))},o}(i.a.Component),ke=window.fetch;window.fetch=function(e,t){(t=t||{}).headers=t.headers||{},t.headers["x-xsrf-token"]=document.body.getAttribute("data-csrf");for(var n=arguments.length,r=new Array(n>2?n-2:0),o=2;o<n;o++)r[o-2]=arguments[o];return ke.apply(void 0,[e,t].concat(r))},Object(de.render)(i.a.createElement(ee,null,i.a.createElement(Z,null,i.a.createElement(q,{exact:!0,path:"/"},i.a.createElement(V,{to:"/pattern/client"})),i.a.createElement(q,{path:"/pattern/:pattern",component:Ee}),i.a.createElement(q,{path:"*"},i.a.createElement(V,{to:"/pattern/client"})))),document.getElementById("app"));var Te=window.alert;window.alert=function(){for(var e=arguments.length,t=new Array(e),n=0;n<e;n++)t[n]=arguments[n];setTimeout((function(){Te.call.apply(Te,[window].concat(t))}),500)}}]);
```

Comment:
Having trouble seeing what this diff is - what does this change do?

Reply:
Good question. This paypal-checkout-demo repo has a manual process for building the client-side assets. This file got updated from running `npm run build`. Other projects automatically do this as part of the publish step.